### PR TITLE
Definition of Done for Engineer

### PR DESCRIPTION
**Pull request by jorg3lopez**
# Def. of Done for Engineers
This PR documents that the "definition of done", for engineering, under the story 43, has reached an agreement and a check mark has been added to symbolize completion. At the same time, a definition of done for "Research & Delivery" has been added. 

#### Changes
Added check mark to definition of done (engineering) in story 43

## Issue
- Definition of done under story 43.
- Added new check box field for "Research & Delivery"

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/story.md [hunk 1]
```diff
@@ -29,4 +29,5 @@ Add any other context or screenshots about the story here.
 
 ## Definition of Done
 
-- [ ] _TBD._
+- [x] Engineering
+- [ ] Research & Discovery
```
Comment (jorg3lopez) [inline .github/ISSUE_TEMPLATE/story.md:33]: We still don't know if `Research & Delivery` want a definition of done.

**Review by tjohnson7021 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/story.md [hunk 2]
```diff
@@ -29,4 +29,5 @@ Add any other context or screenshots about the story here.
 
 ## Definition of Done
 
-- [ ] _TBD._
+- [x] Engineering
```
Comment (halprin) [inline .github/ISSUE_TEMPLATE/story.md:32]: We should actually have our definition of done list in here.  E.g.
```markdown
- [ ] Threat model updated
- [ ] Code refactored for clarity and no design/technical debt
- [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies.
```
Etcetera.

**Review by halprin — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/story.md [hunk 3]
```diff
@@ -29,4 +29,23 @@ Add any other context or screenshots about the story here.
 
 ## Definition of Done
 
-- [ ] _TBD._
+### Engineering
+- [ ] Threat model updated
+- [ ] Code refactored for clarity and no design/technical debt
+- [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies.
+- [ ] Source code is merged to the main branch.
+- [ ] nit test coverage of our code >= 90%
```
Comment (halprin) [inline .github/ISSUE_TEMPLATE/story.md:37]:
```suggestion
- [ ] Unit test coverage of our code >= 90%
```

**Review by halprin — COMMENTED** [review]
Sorry, just one small spelling nit.

**Review by halprin — APPROVED** [review]
Oh yeah!

File: .github/ISSUE_TEMPLATE/story.md (hunk 2 above)
Comment (jorg3lopez) [reply .github/ISSUE_TEMPLATE/story.md]: I like that idea, good point.

**Pull request merged**